package-lock.json

```diff
@@ -77,7 +77,7 @@
     },
     "@types/filewriter": {
       "version": "0.0.28",
-      "resolved": "http://registry.npmjs.org/@types/filewriter/-/filewriter-0.0.28.tgz",
```

Comment:
wow why were these http and not https? strange.

Reply:
It kept changing from https to http to some people.

Reply:
rebase to master again, this change shouldn't be necessary.